### PR TITLE
ANDROID-11446 Deleting Mistica-compose library

### PR DESCRIPTION
**Pull request by gmerinojimenez**
### :goal_net: What's the goal?
Deleting Mistica library and including all the composables inside `mistica` library
### :construction: How do we do it?
Mostly moving packages and deleting the mistica-library publication code

### ☑️ Checks
- [X] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.

### CODE REVIEW

File: library/src/main/java/com/telefonica/mistica/compose/README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ There is a Mística-compose catalog in the [Demo app](https://github.com/Telefon
 | Title                     | ✅  | Composable    |
 
 ***Note**: Not every text input type have been migrated, see [text input](https://github
-.com/Telefonica/mistica-android/tree/master/library-compose/src/main/java/com/telefonica/mistica/compose/input/README.md) section to see which ones are migrated.
+.com/Telefonica/mistica-android/tree/master/library/src/main/java/com/telefonica/mistica/compose/input/README.md) section to see which ones are migrated.
```
Comment (dagonco) [inline library/src/main/java/com/telefonica/mistica/compose/README.md:120]: Here we're setting `master` branch. Probably, if someday change to `main` there is a chance that we could forgot rename it.

For me, seems like a nice idea to be agnostic to the main branch. WDYT?

**Review by dagonco — APPROVED** [review]
Great @gmerinojimenez! Nice job!

File: library/src/main/java/com/telefonica/mistica/compose/README.md (hunk 1 above)
Comment (gmerinojimenez) [reply library/src/main/java/com/telefonica/mistica/compose/README.md:120]: totally, I'll change it

**Review by yamal-alm — APPROVED** [review]
thanks a lot 👏

File: library/src/main/java/com/telefonica/mistica/compose/README.md (hunk 1 above)
Comment (dagonco) [reply library/src/main/java/com/telefonica/mistica/compose/README.md:120]: I don't know why but I thought it wasn't necessary... 🤦🏻‍♂️

Using first option the link wont be broken but "outdated". So master is nice, @gmerinojimenez no need to do anything to this file. Sorry!

Comment (gmerinojimenez) [reply library/src/main/java/com/telefonica/mistica/compose/README.md:120]: anyway, you had a point, thanks both of you for the info :)

**Pull request merged**